apps/fabric-website/src/pages/PageStyles.module.scss

```diff
@@ -1,3 +1,23 @@
+@import '../styles/variables';
+
+.pageTypography {
```

Comment:
I'd rename this to something like `.basePage`. There may be cases in the future where you'd want to add styles/overrides to nested, non-typographic elements like this.